specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/readme.csharp.md

```diff
@@ -0,0 +1,18 @@
+# Cognitive Services Language SDK for .NET
```

Comment:
This file should be merged with the one directly under the "Language" folder - not here.

Reply:
fixed